### PR TITLE
doc: Add the MQTT integration guide for Eclipse Paho client

### PR DESCRIPTION
**Pull request by nejraselimovic**
#### Summary
Adding the documentation for integration with Eclipse Paho.
Reference: https://github.com/TheThingsNetwork/lorawan-stack/issues/3106

#### Testing
Tested with Paho client running on a local machine and a CH TTS deployment.

#### Checklist
- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md` for the chosen target branch.
- [x] Documentation: Relevant documentation is added or updated.
- [ ] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [ ] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

**Review by benolayinka — CHANGES_REQUESTED** [review]
Please consolidate the sections in to one, with #Publishing Downlink Messages and #Subscribing to Upstream Traffic sections. I did this for https://github.com/TheThingsNetwork/lorawan-stack/pull/3148#pullrequestreview-483403952 as an example

**Comment by nejraselimovic** [comment]
> Please consolidate the sections in to one, with #Publishing Downlink Messages and #Subscribing to Upstream Traffic sections. I did this for [#3148 (review)](https://github.com/TheThingsNetwork/lorawan-stack/pull/3148#pullrequestreview-483403952) as an example

Done, please approve if that's all

File: doc/content/integrations/mqtt-clients/eclipse-paho/_index.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    print(a.topic)
+    print(a.payload)
+```
+
+Save the file and run it with 
+
+```bash
+$ python subscribe.py
+```
+command in the terminal.
```
Comment (benolayinka) [inline doc/content/integrations/mqtt-clients/eclipse-paho/_index.md]:
```suggestion
```

File: doc/content/integrations/mqtt-clients/eclipse-paho/_index.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+```
+
+Save the file and run it with 
+
+```bash
+$ python subscribe.py
+```
+command in the terminal. 
+
+The list containing `msg_count` messages published during the default 60 seconds period will be shown in the terminal as a result of running this script.
```
Comment (benolayinka) [inline doc/content/integrations/mqtt-clients/eclipse-paho/_index.md]:
```suggestion
Running this script will show the most recent `msg_count` messages published during the last 60 seconds.
```

File: doc/content/integrations/mqtt-clients/eclipse-paho/_index.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+In case you want to use TLS for security, change the port value to `8883` and pass the `tls` argument to the `simple` function according to its [definition](https://pypi.org/project/paho-mqtt/#id4), where `tls` has to at least contain the path to the CA certificate for your deployment.
```
Comment (benolayinka) [inline doc/content/integrations/mqtt-clients/eclipse-paho/_index.md]:
```suggestion
To use TLS for security, change the port value to `8883` and pass the `tls` argument to the `simple` function according to its [definition](https://pypi.org/project/paho-mqtt/#id4), where `tls` has to at least contain the path to the CA certificate for your deployment.
```

File: doc/content/integrations/mqtt-clients/eclipse-paho/_index.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+publish.single("v3/{application-id}/devices/{device-id}/down/push", '{"downlinks":[{"f_port": 15,"frm_payload":"vu8=","priority": "NORMAL"}]}', hostname="thethings.example.com", port=1883, {'username':"app1",'password':"NNSXS.VEEBURF3KR77ZR.."})
+```
+
+Save the file and run it the terminal with:
+
+```bash
+$ python publish.py
+```
+
+You will see the downlink message being scheduled in {{% tts %}} Console under **Live data** tab and your end device will receive the message in a short time.
```
Comment (benolayinka) [inline doc/content/integrations/mqtt-clients/eclipse-paho/_index.md]:
```suggestion
You will see scheduled messages in the console under the **Live data** tab and your end device will receive the message after a short time.
```

**Review by benolayinka — APPROVED** [review]

**Pull request merged**